test/Utilities/universalfallback.jl

```diff
@@ -113,17 +113,21 @@ y, z = MOI.add_variables(uf, 2)
 end
 @testset "Constraint Attribute" begin
     global x, y, z
+    _saf(vi) = convert(MOI.ScalarAffineFunction{Float64}, MOI.SingleVariable(vi))
+    function _add_constraint(vi, ub)
+        return MOI.add_constraint(uf, _saf(vi), MOI.LessThan(ub))
+    end
     attr = MOIT.UnknownConstraintAttribute()
     @testset "Supported constraint" begin
-        cx = MOI.add_constraint(uf, x, MOI.LessThan(0.))
-        cy = MOI.add_constraint(uf, y, MOI.LessThan(1.))
-        cz = MOI.add_constraint(uf, z, MOI.LessThan(2.))
+        cx = _add_constraint(x, 0.)
```

Comment:
An underscore isn't a license to stop using good names :).
What about `add_saf_constraint`? (I'm uncomfortable with the `saf` abbreviation, but this would at least be an improvement.)